submissions/adeyemiJuwon/todo/src/components/TodoList.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+export default class TodoList extends Component {
+  render() {
+    return (
+      <li complete={this.props.todo.complete}>
+        <label
```

Comment:
Why are you using label?

Reply:
Nothing really. I just think because I am using checkbox input, let me make it as a label instead of using h or p tag

Reply:
You could use `span` or `p`, not label. That isn't what the tag attribute should be used for.